packages/frontend/tests/integration/components/instructor-group/courses-test.js

```diff
@@ -43,11 +43,11 @@ module('Integration | Component | instructor-group/courses', function (hooks) {
     await render(hbs`<InstructorGroup::Courses @instructorGroup={{this.instructorGroup}} />`);
     assert.strictEqual(component.title, 'Associated Courses (3)');
     assert.strictEqual(component.courses.length, 3);
-    assert.strictEqual(component.courses[0].text, 'course 0');
+    assert.strictEqual(component.courses[0].text, 'course 0 (2013)');
     assert.strictEqual(component.courses[0].url, '/courses/1');
-    assert.strictEqual(component.courses[1].text, 'course 1');
+    assert.strictEqual(component.courses[1].text, 'course 1 (2013)');
     assert.strictEqual(component.courses[1].url, '/courses/2');
-    assert.strictEqual(component.courses[2].text, 'course 2');
```

Comment:
I'd like to see a second test added that uses the alternative config and outputs the xxxx - xxxx format. Searching for the config name in tests may get you something to copy and paste from as an example for how to use mirage to setup a different config option.